src/runtime_src/tools/scripts/petalinux.sh

```diff
@@ -59,9 +59,17 @@ echo " * Setup PetaLinux: $PETALINUX_LOCATION"
 
 # dsa_build.sh will create ${PLATFORM_NAME}/ directory for petalinux project
 if [ -z $BSP ]; then
-	${XRT_REPO_DIR}/src/runtime_src/tools/scripts/peta_build.sh ${ORIGINAL_DIR}/dsa_build/${PLATFORM_NAME}.dsa
+	if [ -d "${XRT_REPO_DIR}/src/platform/${PLATFORM_NAME}/config.sh" ]; then
```

Comment:
This is not necessary, if the config.sh exist in the .dsa directory, the peta_build.sh would source config.sh.

Reply:
code updated